carbonmark-api/src/utils/helpers/fetchMarketplaceListings.ts

```diff
@@ -37,26 +45,11 @@ export const getCreditListings = async (
   return project;
 };
 
-/**
- * Query the subgraph for active marketplace listings and project data for the given project
- * Filters out deleted, sold-out and inactive listings
- * Fetches seller profile info from firebase
- */
-export const fetchMarketplaceListings = async (
-  sdk: GQL_SDK,
-  { key, vintage, expiresAfter, fastify }: Params
-): Promise<[Listing[], Activity[]]> => {
-  const { project } = await sdk.marketplace.getProjectById({
-    projectId: key + "-" + vintage,
-    expiresAfter: expiresAfter ?? String(Math.floor(Date.now() / 1000)),
-  });
-  const filteredListings = project?.listings?.filter(isActiveListing) || [];
-  const filteredActivities =
-    project?.activities?.filter(filterUnsoldActivity) || [];
-
-  const formattedListings = filteredListings.map(formatListing);
-
-  const formattedActivities = filteredActivities.map((activity) => ({
+const formatActivities = async (
```

Comment:
This function is also fetching user data from firebase and mapping it. 


```suggestion
const mapUserToActivities = async (
```

Reply:
Sorry, I cannot find the `mapUserToActivities` function in the source code.

Reply:
Ok my second synaps connected... Done in 3ed24c00e